101-perforce-helix-core-server/azuredeploy.json

```diff
@@ -0,0 +1,357 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+          "description": "The base URI where artifacts required by this template are located including a trailing '/'"
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-perforce-helix-core-server/"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+          "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated. Use the defaultValue if the staging location is not secured."
+      },
+      "defaultValue": ""
+    },
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "The location where these resources will be deployed."
+      },
+      "defaultValue": "[resourceGroup().location]"
+    },
+    "OS": {
+      "type": "string",
+      "allowedValues": ["Ubuntu 18.04 LTS", "CentOS 7.5", "RHEL 7.x"],
+      "metadata": {
+        "description": "The operating system of the VM."
+      },
+      "defaultValue": "CentOS 7.5"
+    },
+    "VMSize": {
+      "type": "string",
+      "metadata": {
+        "description": "Please select the size of the VM you wish to deploy.  Read more about sizing options here: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-general"
+      },
+      "defaultValue": "Standard_B2s"
+    },
+    "dataDiskSize": {
+      "type": "int",
+      "minValue": 10,
+      "maxValue": 2048,
+      "metadata": {
+        "description": "Please select the size of the data disk you wish to deploy (value is integer GB) to hold your metadata, logs and depot (archive files). This can be any value up to 2TB (2048 GB)."
+      },
+      "defaultValue": 50
+    },
+    "p4Port": {
+      "type": "int",
+      "metadata": {
+        "description": "P4PORT value to connect to server (via ssl)"
+      }
+    },
+    "swarmPort": {
+      "type": "int",
+      "metadata": {
+        "description": "Port for Swarm (Apache) to be setup on."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username for Virtual Machine"
+      }
+    },
+    "adminSSHPubKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "SSH Public Key for the Virtual Machine."
+      }
+    },
+    "source_CIDR": {
```

Comment:
add an option (and default for "*")

Reply:
Not sure what you mean by 'option'. Have added default.